### PR TITLE
v2.5 release note for visibility tables

### PR DESCRIPTION
**Pull request by manuwelakanade-GSLAB**
v2.5 release note for visibility tables

### CODE REVIEW

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Can you please review/merge this pull request. Thanks.

File: docs/release-notes.rst [hunk 1]
```diff
@@ -55,6 +55,17 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
 
 ------
 
+Visibility
+----------
+
+**Visibility report downloads do not work in Chrome browser version 83 and higher**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:61]: ...do not work in the Google Chrome browser version 83 and higher

File: docs/release-notes.rst [hunk 2]
```diff
@@ -55,6 +55,17 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
...
+
+F5 ACI ServiceCenter's Visibility tab has a **Download Report** icon for downloading VLAN, VIP and Node table data in CSV format. This download functionality does not work in Chrome browser version 83 and higher, due to a new security check added by Chrome: https://developers.google.com/web/updates/2020/04/chrome-83-deps-rems#disallow_downloads_in_sandboxed_iframes
```
Comment (JosephPJordan) [inline docs/release-notes.rst:63]: ... does not work in Chrome version 83 and higher (or 'the Chrome browser version 83 and higher')

File: docs/release-notes.rst [hunk 3]
```diff
@@ -55,6 +55,17 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
...
 ------
 
+Visibility
+----------
+
+**Visibility report downloads do not work in Chrome browser version 83 and higher**
+
+F5 ACI ServiceCenter's Visibility tab has a **Download Report** icon for downloading VLAN, VIP and Node table data in CSV format. This download functionality does not work in Chrome browser version 83 and higher, due to a new security check added by Chrome: https://developers.google.com/web/updates/2020/04/chrome-83-deps-rems#disallow_downloads_in_sandboxed_iframes 
+
+**Workaround:** Use a different browser like Mozilla Firefox or use a lower version of Google Chrome, to download the visibility reports.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:65]: To download visibility reports, use a different browser, like Mozilla Firefox, or an earlier version of Google Chrome.

**Comment by JosephPJordan** [comment]
Looks good @manuwelakanade-GSLAB, just a couple suggested changes

**Comment by manuwelakanade-GSLAB** [comment]
Thanks @JosephPJordan, I have addressed the comments.

**Pull request merged**